test/catch2/block/api/test_block_api_misc.cpp

```diff
@@ -37,8 +37,11 @@ static void
 check_bm_stats(WT_SESSION_IMPL *session, WT_BM *bm)
 {
     WT_DSRC_STATS stats;
+    uint32_t stat_flags;
 
-    // Enable statistics on the connection to allow the statistic macros to update correctly.
+    // Save and Enable statistics on the connection to allow the statistic macros to update
+    // correctly.
+    stat_flags = S2C(session)->stat_flags;
```

Comment:
Why is this done? The macros should work correctly if the flags are set correctly.

Reply:
Why - To show that flags are needed.
The alternative is to delete it. I am doing that now.